packages/dashboard/src/widget-resize-controller.js

```diff
@@ -138,13 +139,31 @@ export class WidgetResizeController {
   }
 
   /** @private */
-  __updateResizedItem(colspan = 1, rowspan = 1) {
-    if ((this.resizedItem.colspan || 1) === colspan && (this.resizedItem.rowspan || 1) === rowspan) {
+  __updateResizedItem(colspanDelta, rowspanDelta) {
+    this.__resizeItem(this.resizedItem, colspanDelta, rowspanDelta);
+    requestAnimationFrame(() => this.__updateWidgetStyles());
+  }
+
+  /** @private */
+  __resizeItem(item, colspanDelta, rowspanDelta) {
+    if (item.items) {
+      // Do not resize sections
       return;
     }
 
+    const gridStyle = getComputedStyle(this.host.$.grid);
+    const columns = gridStyle.gridTemplateColumns.split(' ');
+    const newColspan = Math.min(Math.max((item.colspan || 1) + colspanDelta, 1), columns.length);
```

Comment:
Nit (optional, feel free to discard): readability might be enhanced by using two extra constants:

```
  const currentColspan = item.colspan || 1;
  const currentRowspan = item.rowspan || 1;
  
  const newColspan = Math.min(Math.max(currentColspan + colspanDelta, 1), columns.length);
  const newRowspan = Math.max(currentRowspan + rowspanDelta, 1);

...
  ```

Reply:
Done